Vagrantfile

```
@@ -5,6 +5,10 @@ Vagrant.configure("2") do |config|
   config.vm.box = "centos/stream9"
   config.vm.synced_folder ".", "/vagrant"
 
+  config.vm.provision "ansible" do |ansible|
+    ansible.playbook = "development/playbooks/etc_host.yml"
```

```suggestion
    ansible.playbook = "development/playbooks/etc_host.yml"
    ansible.compatibility_mode = "2.0"
```

this fixes the network/dns issue

---

The problem is that Vagrant does not properly detect the version of Ansible, and falls back to Ansible 1.8 compatibility mode. However, in that mode, the inventory ends up with only a single host and using 127.0.0.1 IPs.

---

How'd you figure that out?

---

I used https://github.com/marketplace/actions/debugging-with-ssh to ssh into the GitHub Actions runner and then ran `vagrant destroy client` and `vagrant up client`, saw the "Using Ansible 1.8 compat" warning, and was enlightened.